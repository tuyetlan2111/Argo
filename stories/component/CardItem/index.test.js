import React from 'react';
import renderer from 'react-test-renderer';
import { CardItem } from './index';

const data = {
  id: 1,
  category: 'SUV',
  title: '2022 Kia Suv',
  imageSrc: 'https://e7.pngegg.com/pngimages/114/496/png-clipart-kia-motors-car-2016-kia-sportage-2018-kia-sportage-kia-compact-car-car.png',
  price: 23805,
  btnText: 'See Details',
  description: 'aaaa'
}

it('test props', () => {
  const component = renderer.create(
    <CardItem card={data} />
  ).toJSON()

  expect(component).toMatchSnapshot();
});
