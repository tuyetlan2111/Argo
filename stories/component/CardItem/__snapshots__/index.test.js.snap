// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test props 1`] = `
<div
  className="card-item"
>
  <div
    className="category"
  >
    SUV
  </div>
  <h3
    className="title"
  >
    <a
      href=""
    >
      2022 Kia Suv
    </a>
  </h3>
  <div
    className="image"
  >
    <img
      alt="image"
      src="https://e7.pngegg.com/pngimages/114/496/png-clipart-kia-motors-car-2016-kia-sportage-2018-kia-sportage-kia-compact-car-car.png"
    />
  </div>
  <div
    className="content"
  >
    <span
      className="price"
    >
      $
      23805
    </span>
    <p
      className="description"
    >
      aaaa
    </p>
  </div>
  <button
    className="storybook-button storybook-button--medium storybook-button--secondary"
    style={
      {
        "backgroundColor": "#ec9031",
        "color": "#fff",
      }
    }
    type="button"
  >
    Button
  </button>
</div>
`;
